### PR TITLE
add onmousemove event to temporarily show controls in fullscreen and …

### PR DESCRIPTION
**Pull request by dimola7**
…normal modes

### CODE REVIEW

File: src/components/VideoControls/VideoControls.css [hunk 1]
```diff
@@ -8,6 +8,9 @@
   left: 50%;
   transform: translate(-50%, -50%);
 }
+.controls:-webkit-fullscreen{
+  display: none;
```
Comment (ifedapoolarewaju) [inline src/components/VideoControls/VideoControls.css:12]: is this change needed? Or was it committed by accident? 🤔

File: src/components/VideoControls/VideoControls.css [hunk 2]
```diff
@@ -34,6 +37,8 @@
   background-color: gray;
   display: flex;
   position: absolute;
+  /*margin-left: 10px;*/
+  /*padding-right: 10px;*/
```
Comment (ifedapoolarewaju) [inline src/components/VideoControls/VideoControls.css:41]: we should delete this right?

File: src/components/VideoControls/VideoControls.css (hunk 1 above)
Comment (dimola7) [reply src/components/VideoControls/VideoControls.css:12]: Yes, it was committed by accident, I'd delete it

File: src/components/VideoControls/VideoControls.css (hunk 2 above)
Comment (dimola7) [reply src/components/VideoControls/VideoControls.css:41]: Will delete it
